style.css

```diff
@@ -70,30 +85,27 @@ footer {
   width: 100%;
 }
 
+.text_transform {
+  text-transform: capitalize;
+}
+
 .link_flex_container {
   width: 80%;
   list-style-type: none;
 }
 
-.logo {
+.font_size {
```

Comment:
make this class name descriptive. This font size is definitely for whole document. I guess it's for some kind of heading 🤔